razorpay-subscriptions.php

```diff
@@ -43,6 +43,9 @@
 add_action('plugins_loaded', 'woocommerce_razorpay_subscriptions_init', 20);
 add_action('admin_post_nopriv_rzp_wc_webhook', 'razorpay_webhook_subscription_init', 20);
 
+add_action('activated_plugin', 'razorpaySubscriptionPluginActivated', 10, 2 );
```

Comment:
@sonyy-m remove whitespace before ). and test if any other plugin is activated razorpaySubscriptionPluginActivated should not be processed,